onnxruntime/test/python/onnxruntime_test_python_mlops.py

```diff
@@ -133,7 +133,17 @@ def testLabelEncoder(self):
         np.testing.assert_allclose(output_expected, res[0], rtol=1e-05, atol=1e-08)
 
     def test_run_model_mlnet(self):
-        sess = onnxrt.InferenceSession(get_name("mlnet_encoder.onnx"))
+        available_providers = onnxrt.get_available_providers()
+
+        # The Windows GPU CI pipeline builds the wheel with both CUDA and DML enabled and ORT does not support cases
```

Comment:
i had left a comment in the other review. i guess working this test is okay for us to move forward, but ideally, we should prevent one from being able to create a build which default registers CUDA and DML together.